grammars/python.cson

```diff
@@ -335,7 +335,7 @@
     ]
   }
   {
-    'begin': '\\b(lambda)\\s?+'
+    'begin': '\\b(lambda)(?=[\\s\\:])'
```

Comment:
Would something as simple as a word boundary anchor (`\b`) work here, or do you need the lookahead?

Reply:
That s a very good point! Let me push a new commit